Resources/Textures/_RMC14/Mobs/Avali/avali_parts.rsi/meta.json

```
@@ -1,7 +1,7 @@
 {
-    "version": 1,
+    "version": 1.1,
```

This doesn't need to be changed, it's used to note the version of the rsi format used, not the version of the rsi itself
```suggestion
    "version": 1,
```